src/js/controller/instream-adapter.js

```diff
@@ -164,8 +164,10 @@ define([
             }
 
             if (_array && _arrayIndex + 1 < _array.length) {
-                // fire complete event
-                this.trigger(events.JWPLAYER_MEDIA_COMPLETE, data);
+                // fire complete event if not ad skipped
+                if (e.type !== events.JWPLAYER_AD_SKIPPED) {
+                    this.trigger(events.JWPLAYER_MEDIA_COMPLETE, data);
+                }
```

Comment:
```_instreamItemNext``` is also called when ad is skipped.
The ```customNext``` was created for freewheel, since freewheel handles ad pod by itself.
Updated the PR so that ```_instreamItemNext``` does not fire complete event. Made a separate function that is triggered when instream item completes.